tasks/options/template.js

```diff
@@ -67,13 +67,17 @@ module.exports = function(grunt) {
             options: data('http://localhost:5000/api', true),
             files: files
         },
+        dev: {
+            options: data('http://localhost:5000/api', true),
+            files: {'index.html': '<%= appDir %>/index.html'}
```

Comment:
Yes, this is a bit odd based on our setup, but I couldn't figure out a way to set a `base` in WebPack like we did with `grunt-connect`. The only way I can think of is to set the root in the `dist` folder but that would make all of the scripts unavailable because they would become located at `../script`.

There might be a way though but I didn't want to let this stall me from finishing the rest of the work...